addons/binding/org.openhab.binding.loxone/src/main/java/org/openhab/binding/loxone/LoxoneBindingConstants.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.loxone;
+
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+
+/**
+ * Common constants used across the whole binding.
+ *
+ * @author Pawel Pieczul - Initial contribution
+ */
+public class LoxoneBindingConstants {
+
+    public static final String BINDING_ID = "loxone";
+
+    // List of all Thing Type UIDs
+    public static final ThingTypeUID THING_TYPE_MINISERVER = new ThingTypeUID(BINDING_ID, "miniserver");
+
+    // Channel IDs are created dynamically in runtime
+
+    // Miniserver properties and parameters
+    public static final String MINISERVER_PARAM_HOST = "host";
+    public static final String MINISERVER_PARAM_PORT = "port";
+    public static final String MINISERVER_PROPERTY_SERIAL = "Serial number";
```

Comment:
I did not know about these, I will add population of vendor, model and serial from the discovery into these properties.